packages/rollup-full-node/test/app/web-rpc-handler.spec.ts

```diff
@@ -93,6 +97,42 @@ describe('Web3Handler', () => {
     }
   })
 
+  describe('the getBlockByNumber endpoint', () => {
+    it('should return a block with the correct timestamp', async () => {
+      const httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      const timestamp = await httpProvider.send('evm_getTime', [])
```

Comment:
I actually don't think `evm_getTime` is a valid test RPC method (can't find anything about it on a quick google search, lmk if I missed it).

Reply:
Ah I remember why I added this now! Originally we weren't storing timestamps in blocks. This made it so there was no way  to query the timestamp using existing eth rpc API methods so I had to add a new one. Now that we are it can be removed. -> [Remove 'evm_getTime' handler](https://github.com/ethereum-optimism/optimism-monorepo/pull/59/commits/07ba24099aee9dd3afd24c5e70ce489a242789dc)